Log4j functionality implemented

NikolayAbysov opened the pull request.

NikolayAbysov committed:
Log4j functionality implemented

RomanDubovskyi commented on .gitignore:
```diff
@@ -1,3 +1,4 @@
-target/*
\ No newline at end of file
+target/*
+logs/*
```
потрібно додати пусту строку в кінець файлу

NikolayAbysov replied:
Cool! Thanks!

NikolayAbysov committed:
Issues fixed:
1 All URL's added to filter
2 List changed to Set in User model
3 Other minor changes
